src/chapters/razor-views/using-templates.rst

```diff
@@ -0,0 +1,90 @@
+Using a Template
+================
+
+This page and the ones that follow introduce how to start to pass data between the MVC elements.
+We haven't addressed models yet (those come next) so this just means sharing information between 
+the controller and view. 
+
+.. index:: ! ViewBag
+
+Passing Data to a Template
+--------------------------
+
+The controller class contains methods that send data to different templates.
+These methods have a structure similar to:
+
+.. sourcecode:: c#
+   :linenos:
+
+   public IActionResult ActionMethod()
+   {
+      // method code here
+      
+      ViewBag.property = Data;
+      return View();
+   }
+
+**ViewBag** is an object that passes data into a template. ``Data`` can
+be a variable of any type, a number, a collection of some sort, or an object. 
+A ``ViewBag`` property is created and given a value as simply as is done on line 5 above.
+
+Accessing Data in a Template
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The data assigned to properties on ``ViewBag`` is available inside of Razor templates.
+It can be accessed with the syntax ``@ViewBag.property``.
+
+For example, if the controller stores data in ``ViewBag.name``, then the
+template can display that value like so:
```

Comment:
It feels like there is a bit of disconnect between this sentence and the preceding one. Maybe expand to "For example, we have a property called ``name`` and the controller is storing some string in ``ViewBag.name``. We could pass that string to the template using the at sign: "

Reply:
I changed some wording around this example and used a different property name. interested to know if the change I made from the comment just before this - on how to create and assign property values - will help to clarify this section also. ``name`` itself is nothing before being created as a property of ``ViewBag``

Reply:
I think the new wording is great! It really helps clarify the connection and it reinforces to me as the reader that I create the properties and I have some sense of ownership over what the properties of ``ViewBag`` are.